airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLBulkLoadCheckTest.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.mssql.v2
+
+import io.airbyte.cdk.load.check.CheckIntegrationTest
+import io.airbyte.cdk.load.check.CheckTestConfig
+import io.airbyte.cdk.load.test.util.FakeConfigurationUpdater
+import io.airbyte.integrations.destination.mssql.v2.config.MSSQLSpecification
+import java.nio.file.Path
+
+class MSSQLBulkLoadCheckTest :
+    CheckIntegrationTest<MSSQLSpecification>(
+        successConfigFilenames =
+            listOf(
+                CheckTestConfig(
+                    Path.of(BulkInsert.CONFIG_FILE),
+                    name = "Bulk Load Check Should work",
+                ),
+            ),
+        emptyMap(),
+        configUpdater = FakeConfigurationUpdater,
```

Comment:
for my understanding: we can't just put this config into the MSSQLCheckTest, because we need a different config updater?

(wondering if it would make sense to put the configUpdater into the CheckTestConfig)

Reply:
Yes! For the INSERT LOAD tests we need to update the config with the information from the mssql container but for the BULK load tests since we are running it against an actual sql server instance, we dont need it and can use the secret value from GSM as is